integration-tests/src/test/resources/application.yml

```diff
@@ -15,14 +15,14 @@ flusswerk:
   routing:
     exchange: my.exchange
     incoming:
-      - first
-      - second
+      - input.first
+      - input.second
     outgoing:
-      default: target.queue
+      default: output
     failure policies:
-      first:
+      "[input.first]":
```

Comment:
I recall you stumbled upon that syntax while developing the tests, maybe this is something that should  be mentioned in the README? I.e. a small `NOTE: If your queue names have dots in them, you have to double-quote and wrap them in square brackets (e.g. "[input.first]") when using them as keys (e.g. in failure policies).`

Reply:
I've added some extra documentation and an example.